neqo-common/src/datagram.rs

```diff
@@ -160,19 +160,19 @@ impl<D: AsRef<[u8]>> AsRef<[u8]> for Datagram<D> {
 /// Upholds Linux GSO requirement. That is, all but the last datagram in the
 /// batch have the same size. The last datagram may be equal or smaller.
 #[derive(Clone, PartialEq, Eq)]
-pub struct DatagramBatch {
+pub struct Batch {
     src: SocketAddr,
     dst: SocketAddr,
     tos: Tos,
     datagram_size: NonZeroUsize,
     d: Vec<u8>,
 }
 
-impl Debug for DatagramBatch {
+impl Debug for Batch {
     fn fmt(&self, f: &mut Formatter) -> fmt::Result {
         write!(
             f,
-            "DatagramBatch {:?} {:?}->{:?} {:?}: {}",
+            "Batch {:?} {:?}->{:?} {:?}: {}",
```

Comment:
Is this a change you want?  Debug statements don't have module context, so it might be better to say "Datagrams" or similar.